packages/database/src/prisma-api.ts

```diff
@@ -424,9 +424,9 @@ export function getSdk(
       })
 
       if (coupon) {
-        return await ctx.prisma.$transaction([merchantCharge, purchase, coupon])
+        return await ctx.prisma.$transaction([purchase, merchantCharge, coupon])
```

Comment:
I moved these around because purchase is the only thing used on the call side was "purchase"